README.md

```diff
@@ -21,7 +21,7 @@ Commit naming follows the [Conventional Commits](https://www.conventionalcommits
 \[optional footer]
 ```
 
-The type should be chosen in these : feat, fix, chore, ci, docs, style, refactor, perf, test, revert
+The type should be chosen in these : feat, fix, build, chore, ci, docs, style, refactor, perf, test, revert
```

Comment:
Hi, this change is needed, since in the future we'll be adding many more libraries to our project. Appart from a minor spelling mistake that I corrected in 2027e496942d386237d7694eb1d619993f4e47be for your, this is a fine addition to our codebase.